### DISCUSSION TITLE
Redirect between pages

### DISCUSSION THREAD

**Discussion by f33r0** [vuejs/vitepress#2542] [Q&A]
Hello. Is it possible to add or use redirection between pages in vitepress?

**Comment by roma-marshall** [comment]


**Reply by brc-dd** [reply]
Hey, avoid posting AI generated answers unless you've verified that they are correct. vue-router won't work properly with vitepress. The proper way is to just write a plain markdown/html link. If you want to do that programmatically, you can use the [`useRouter` composable](https://vitepress.dev/reference/runtime-api#userouter).

**Reply by f33r0** [reply]
I have too many links...

**Reply by brc-dd** [reply]
@f33r0 Can you explain what you're trying to do?

**Reply by f33r0** [reply]
I need to move all users from site.com/docs/* to the same page without /docs/

**Reply by brc-dd** [reply]
@f33r0 You need to configure that at your hosting service. For Netlify, the configuration will look something like this:

```toml
[[redirects]]
  from = "/docs/*"
  to = "/:splat"
  status = 301
  force = true
```

**Reply by f33r0** [reply]
Thanks for the tip, but I'm using gh pages. Can html be used in ts configurations?

**Reply by javierjulio** [reply]
Same, I'm using GitHub Pages to host a VitePress site. Is it possible to do redirects for a GitHub Pages deployment? The repository using VitePress had its docs using Jekyll so I'd like to redirect old paths to new ones.

**Reply by Zhengqbbb** [reply]
GitHub page doesn't provide 301/304 redirect feature
